### DISCUSSION TITLE
Besoin d'informations et d'aide

### DISCUSSION THREAD

**Discussion by Kjbk73** [TP-Lab/Mobile-SDK#44] [Q&A]
J'ai découvert un portefeuille observé sur mon compte Token pocket sur lequel se trouve des jetons usdt, pouvez vous m'informer à quoi sa sert ce portefeuille et comment l'utiliser.

**Comment by Issalk** [comment]
> J'ai découvert un portefeuille observé sur mon compte Token pocket sur lequel se trouve des jetons usdt, pouvez vous m'informer à quoi sa sert ce portefeuille et comment l'utiliser.

**Reply by COMbaohe** [reply]
可以赚钱
